osquery/tables/utility/file.cpp

```diff
@@ -99,6 +127,171 @@ void genFileInfo(const fs::path& path,
     r["type"] = "unknown";
   }
 
+#else
+
+  // Get the handle of the file object.
+  hFile = CreateFile(TEXT(path.string().c_str()),
+                     GENERIC_READ,
+                     FILE_SHARE_READ,
+                     NULL,
+                     OPEN_EXISTING,
+                     FILE_ATTRIBUTE_NORMAL,
+                     NULL);
+
+  // Check GetLastError for CreateFile error code.
+  if (hFile == INVALID_HANDLE_VALUE) {
+    return;
+  }
+
+  // Get the owner SID of the file.
+  dwRtnCode = GetSecurityInfo(hFile,
```

Comment:
nit: Same here, I'll often use something like `auto ret = `, and then re-use the `ret` value throughout.